src/control-plane/backend/lambda/api/store/aws/quicksight.ts

```diff
@@ -169,7 +182,7 @@ export const registerEmbeddingUserByRegion = async (region: string) => {
       Email: QUICKSIGHT_EMBED_NO_REPLY_EMAIL,
       IamArn: QuickSightEmbedRoleArn,
       Namespace: QUICKSIGHT_NAMESPACE,
-      UserRole: UserRole.READER,
+      UserRole: UserRole.ADMIN,
```

Comment:
Is it the mandatory permission for generating the embedding url?

Reply:
Both `ADMIN` and `Author` users in QuickSight will cost $20 per month. Should we create one user for those two use scenarios?